desc/profiles.py

```diff
@@ -592,9 +592,11 @@ class PowerSeriesProfile(Profile):
 
     _io_attrs_ = Profile._io_attrs_ + ["_basis", "_transform"]
 
-    def __init__(self, params=[0], modes=None, grid=None, sym="auto", name=""):
+    def __init__(self, params=None, modes=None, grid=None, sym="auto", name=""):
```

Comment:
See here for reasoning https://github.com/PlasmaControl/DESC/pull/448#discussion_r1230140709